### DISCUSSION TITLE
GPT-4 因人變笨？

### DISCUSSION THREAD

**Discussion by Eileen050824** [1101-GS4719-C/Python-Community-GS4719-C#1931] [General]
最近GPT-4「變笨」、「變懶」不絕於耳，我也發現GPT-4這個昔日的大聰明回答時逐漸失去理解力和準確性，不時冒出牛頭不近對馬嘴的答案甚至不回答。
我查了下GPT-4降智的原因，加州大學聖克魯茲分校的論文曾提出一種學術界解釋：大模型在「見過」的任務表現優秀，新任務相對不佳。我的解讀是這更像檢索模擬智慧法，回答問題全靠硬記，而非純基於學習理解能力。結合論文所述，「任務污染」造就了許多模型處理早期資料時優異的表現，給你一種「GPT初次回答就這麼得心應手」的美好錯覺，但其實訓練過程AI就「看過」這些資料了。
或許有人悲觀表示：降智說不定是所有大模型的共同命運。不過我認為持續學習是生物神經網路的優勢，學習不同任務可進一步增強系統性能，從一任務獲得的知識有助提升整個學習效率。
然而，我覺得有點矛盾的現實是，現在我們越來越依賴AI產生內容，用退化中大模型答案解決生活實際問題，將來大模型取得的資料，或越來越多是AI創造的東西，而不是來自人類。AI用AI生產物再訓練，最終結果會走向何方？如果不從根本上解決數據污染和持續學習力問題，未來世界很可能會和大模型一樣越變越笨了！
資料來源：品玩、Technews科技新報

**Comment by happy246810** [comment]
我認同前半段GPT變笨的這件事實，因為GPT本身就是將人們給的答案作分析，並給予詢問者一個最貼近的答案，但對於後者世界可能會越變越笨，我是持反對的，因為人是有認知能力的，他能判斷GPT是否能回答他的問題，而持續的得不到答案，也會降低使用的意願，使GPT漸漸地退出市場。但還是感謝版主發出這篇文章，引起關於GPT的討論。

**Comment by KH1126** [comment]
最近使用 ChatGPT 時，我觀察到生成的內容與我提供的描述有些許不同。這種感覺是近期才開始出現的，可能是由於以前主要使用 ChatGPT 來幫助我分析文章的大綱，而現在則是嘗試讓它生成如何寫一個程式作業的架構。這些程式作業可能涉及到更專業、更高深的主題，而這些主題可能不在 ChatGPT 的訓練範圍內，因此它給人的感覺是越來越無法理解、產生的內容也越來越差。

**Comment by 20050201Shawn** [comment]
在機械加工業中，有一名叫鏟花的技術。依照機械的母性原則，機具的機械精度會一代不如一代，因此透過鏟花，在組裝機具前，只能人工削平金屬平面矯正誤差，且目前還未被機器取代。或許我們未來會需要gpt審查員(?)，手動過濾一些垃圾資訊，當然這只是我的淺見，我不認為gpt會越變越笨。

**Comment by SusuTima** [comment]
這學期的有一堂課老師就有在課堂說，我們可以參考網路資料，但是他也會請助教在網路上故意丟一些錯誤的資料，以免我們無腦用gpt去完成所有作業。

**Comment by benbebobo149** [comment]
我是認為說GPT-4應該只是找到不實的資訊導致在回答我們的時候比較笨，因為它搜索的網路上的資訊沒有經過篩檢，直接塞到其模型中來回復你。 我覺得可以等到openAI找到一個比較良好的搜索方法後，在轉回去使用GPT也不急。 我蠻推薦你使用Phind 的，他們就是有限制模型只能從比較權威的網站上面找答案，給出的答案也比較精確。我認為這樣的限制性搜索方法有助於降低不實資訊的傳播，提高回答的準確性和可信度。對於我們這樣依賴人工智慧的時代，這樣的工具非常寶貴。最後，我要感謝同學的分享，這讓我對不同的搜索工具有了更深入的了解。

**Comment by lyykey** [comment]
最近關於GPT-4「變笨」、「變懶」的討論確實很多，這也引起了我對於AI發展的一些思考。我發現GPT-4似乎在遇到新的任務時失去了以往的靈活性和準確性，這讓我感到有些困惑和擔憂。

**Comment by lyykey** [comment]
我理解到這種「降智」現象可能與模型的訓練方式有關，而不僅僅是GPT-4本身的問題。訓練過程中的「任務污染」可能使得模型對於特定任務的處理能力過於依賴先前的訓練資料，而缺乏對新任務的靈活應對能力。

**Comment by lyykey** [comment]
雖然一些人對於大模型的「降智」表示悲觀，但我仍然相信持續學習是生物神經網路的優勢。通過不斷學習和適應不同的任務，AI可以不斷提升自己的性能，這是一個值得期待的方向。

**Comment by lyykey** [comment]
我意識到了一個矛盾的現實，那就是我們日益依賴AI來處理生活中的實際問題，而這可能會加劇數據污染和持續學習力問題。如果我們不解決這些根本性的問題，未來可能會面臨更大的挑戰，甚至出現整個系統變得越來越笨的情況。

**Comment by lyykey** [comment]
我認為我們需要更加重視AI的訓練方式，並努力解決數據污染和持續學習力問題，以確保AI能夠持續發揮其潛力，為人類帶來更多的益處，而不是成為我們的一個負擔。

**Comment by hbh10601** [comment]
自從ChatGPT出來之後，許多作業都參考他而得到答案，很多程式問題都可以透過他得到解決方案，但是其他方面的問題，答案卻不一定精確，尤其是和時事相關的。那時候才知道它其實背後有一個龐大的資料庫，而不是像google真的去搜尋，所以沒看過的東西他就可能會胡謅。而針對4版本有沒有變笨，個人比較沒有感受到，比較多的可能是因為越來越多人使用，生成答案時變慢了許多。

**Comment by wanka2030** [comment]
與其說是GPT變笨之外，更大的原因有可能是開發公司讓GPT更有人為化的徵兆。不僅是讓GPT能夠不能在短時間給出相對應且直接的回答之外，大多數的回答更需要讓使用者能夠透過問問題的模式以及對英的方式，在每一步的步驟中思考、隨著GPT變聰明。

**Comment by 155Julia** [comment]
我有用過GPT-4！我認為它最大的缺點就是太慢了，相較GPT-3不但慢給出的資訊也不一定更完整正確，但我覺得它最大的優點是可以丟給它檔案，讓它直接讀檔解釋、翻譯、摘要內容，還能夠幫你畫圖，雖然這些Copilot也做得到，但免費版的Copilot有限制對話數量，讀檔內容也沒辦法太大，畫圖我認為也沒有GPT-4更精準實用。

**Comment by jeff110303549** [comment]
畢竟大型預言模型目前還算是新興的科技，對於實際應用上確實有時候還不是那麼完美，不過我覺得再給這個產業一些時間去完善結構、制度、資料庫等方面的技術，GPT一定還有更多的應用能夠造福人類社會

**Comment by Hahahahbla** [comment]
你提到的問題非常具有洞察力，也引發了關於大模型（如GPT-4）的發展和應用的深刻思考。

如果不解決數據污染和持續學習力的問題，AI模型的性能可能會逐漸退化，影響其在實際應用中的效果。然而，通過持續改進技術、提升數據質量和建立更加智能的學習機制，AI技術仍有可能實現更高的智慧水平，帶來更多的創新和應用。

你提出的問題和見解對於未來AI技術的發展具有重要的啟示作用，希望我們能在不斷探討和解決這些問題的過程中，推動AI技術的健康發展。

**Comment by alexender0104** [comment]
我認為GPT 系列模型的訓練和性能與人類的智力水平沒有直接關聯。這些模型是基於大量的文本數據訓練的，它們的表現取決於數據的質量、量和模型的設計。它們的目的是生成自然語言文本和理解語言，而不是與人類智力競爭。因此，GPT-4 不會導致人變笨，它只是一種工具，可以用來幫助人們處理和生成文本信息。

**Comment by 110101501** [comment]
我認為GPT所提供的答案僅能供參考，最終的評斷依據還是要仰賴自己的思考模式。
人使用工具，而非工具使用人。要如何適當地使用此工具，也是學習的一環。

**Comment by lilishh** [comment]
同意文章的最後一段，感覺在gpt出現後，人類越來越依賴它，找個資料也想用，但我擔心的是是否會在長期依賴人工智慧的情況下，人們做事的效率、成果變得越來越差，其實真的沒有學到什麼知識。還是適當使用就好。

**Comment by asd99263fg** [comment]
自從有人gpt之後我發現很多資料跟媒體識讀能力就變得很差，我會直接覺得gpt講得就是百分百正確，但其實數學跟基本一些問題其實gpt都還是會有出錯的地方，而當我們過度相信某個事物就跟少了媒體識讀的人類一樣，忘記要自己找到真理跟尋找正確事物的過程

**Comment by sandyho0307** [comment]
我覺得GPT-4這類大模型確實有其侷限性。它們在訓練過程中「看過」大量資料，因此在處理這些已見過的資料時表現很好，但對於新任務，則可能會出現理解力和準確性下降的問題。這有點像是學生在考試前背誦答案，但如果考題一變，他們就會寫不出來。

**Comment by sunset513** [comment]
我對於大型語言模型的降智現象可以說是十分的擔憂，我在使用這些模型時更得注意這種情形，畢竟這可是我們學生做各種作業的必備武器，模型的數據汙染及學習能力下降對學生而言肯定是一大打擊。我真的無法想像沒有大型與語言模型的日子，我該怎麼度過，我敢說我必定會相當難受。

當然我們也要花時間思考，如果我們太依賴這些模型，導致我們思維僵化、放棄思考。而我們又繼續依賴模型，導致一個惡性循環。而且，這也有可能會讓數據污染的問題變得更加嚴重。

最後謝謝版主分享這篇有意義的文章，讓我思考者個有意義的問題

**Comment by Shang621** [comment]
有些人擔心使用GPT-4等人工智能技術會讓人變笨，因為這些技術能夠輕鬆解決問題和提供資訊，減少了人們思考和學習的機會。然而，這種觀點忽略了技術的輔助作用。GPT-4等人工智能工具能夠提高效率，讓人們有更多時間專注於創造性和高層次的思考。同時，人工智能可以幫助人們接觸到更廣泛的信息和知識，促進學習和理解。如果合理使用，GPT-4不僅不會讓人變笨，反而能成為強有力的學習和工作助手，提升人們的智力和創造力。关键在于如何利用这些技术，避免对它们的过度依赖。

**Comment by 710388** [comment]
沒有吧，感覺在一些細節他有更理解我的需求

**Comment by cassy0201** [comment]
如果問題問的足夠精確我覺得他還是可以提供相對應的資訊，但最後的判斷還是需要依靠自己

**Comment by NcuJerryKao** [comment]
您提到的對於GPT-4「變笨」、「變懶」的現象確實引發了一些深入的思考和討論。以下是一些建設性的建議回覆：
1. 持續改進和優化訓練方式：為了避免大模型降智的問題，開發者可以不斷改進訓練方式，包括提高數據質量、多樣性和量，並設計更有效的學習機制，從而提高模型的理解力和準確性。
2. 引入多任務學習：通過使模型同時處理多個不同任務，可以幫助模型更全面地學習和理解不同類型的問題，提高系統性能和適應性。
3. 加強數據篩選和校準：為了避免數據污染，開發者可以加強數據篩選和校準的工作，確保模型訓練所使用的數據具有高質量和真實性，從而提高模型的準確性和智能性。
4. 促進持續學習和進步：除了模型本身的訓練，也需要通過不斷更新和優化模型來提高其性能，以應對不斷變化的任務和環境，從而確保模型的持續學習和進步。
5. 人工智能的合理使用：在使用人工智能技術時，人們應該理性

**Reply by yubo0oo** [reply]
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by stone971275** [comment]
我自己是覺得，有時候對於GPT提問時，急切的口吻，甚至帶點情緒勒索，會使得效率提高。
此外官網也有提供六可以提高效率
1.指令要清晰 (write clear instructions)
2.提供參照(provide reference text)
3.將複雜任務拆成簡單的子任務 (split complex tasks into simpler subtasks)
4.讓 GPTs 有時間思考 (give GPTs time to “think”)
5.善用外部工具 (use external tools)
6.系統性測試改變 (test changes systematically)

**Comment by poiu6fi** [comment]
GPT-4可能真的有降智問題，若不解決數據污染和學習力，未來AI可能會越來越笨
